src/routes/tools/grade-calculator/+page.svelte

```
@@ -1,41 +1,727 @@
+										<p class="mt-2 text-sm leading-6 text-base-content/70">
+											{#if selectedCourse.ruleset}
+												<strong class="font-semibold text-base-content">
+													{selectedCourse.ruleset.calculationMode === 'points'
+														? 'Points-based.'
+														: selectedCourse.ruleset.calculationMode === 'weighted'
+															? 'Weighted.'
+															: selectedCourse.ruleset.calculationMode === 'percentage'
+																? 'Percentage-based.'
+																: 'Hybrid.'}
+												</strong>
+												{' '}
```

ESLint

[error] Unexpected mustache interpolation with a string literal value.

(svelte/no-useless-mustaches)